### DISCUSSION TITLE
Road quality

### DISCUSSION THREAD

**Discussion by DeeAitch** [organicmaps/organicmaps#692] [Ideas]
Just as Idea and if you have time. Add service like this [ https://play.google.com/store/apps/details?id=com.definitics.roadbouncesocial&hl=en_CA&gl=US]( https://play.google.com/store/apps/details?id=com.definitics.roadbouncesocial&hl=en_CA&gl=US)

**Comment by AVanslov** [comment]
Hi! I’ve developed [veloway.net](https://veloway.net/) - a service where anyone can contribute to assessing road quality using just a smartphone. It allows users to upload photos of bike routes, which are then analyzed by neural networks to map the surface quality and issues in real time. So far, I’ve added only a few roads, but the platform is ready to scale collaboratively.

Unfortunately, the project didn't gain traction in terms of users, but I truly believe the idea is solid and can be very useful. I’d be glad if my work could contribute to a bigger open-source ecosystem.

Would somebody be interested in integrating some of my veloway’s algorithms or logic into Organic Maps?
